src/lib.rs

```diff
@@ -11,10 +11,9 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(feature = "alloc", feature(alloc))]
 #![cfg_attr(feature = "nightly", feature(i128_type))]
+#![cfg_attr(feature = "nightly", feature(cfg_target_feature))]
 #![cfg_attr(feature = "bench", feature(test))]
-#![cfg_attr(all(feature = "nightly", feature = "std"), feature(zero_one))]
 
-#![allow(unused_features)]
```

Comment:
FWIW, this was added originally because whatever ancient version of the rustc I had at the time we first started working on this would say that features were unused that _were_ actually used and I just wanted it to STFU. Glad we can remove it now. :)